Resources/Prototypes/_DEN/Species/Ovinia.yml

```
@@ -0,0 +1,154 @@
+- type: species
+  id: Ovinia
+  name: species-name-ovinia
+  roundStart: true
+  prototype: MobOvinia
+  sprites: MobOviniaSprites
+  defaultSkinTone: "#ffffff"
+  markingLimits: MobOviniaMarkingLimits
+  dollPrototype: MobOviniaDummy
+  skinColoration: Hues
+  maleFirstNames: NamesFirstMale
+  femaleFirstNames: NamesFirstFemale
+  lastNames: NamesLast
+  minHeight: 0.5
+ # defaultHeight: 0.9 #Disabled due to EE bug
```

EE bug in wizden upstream? Uncomment and remove note 
```suggestion
  defaultHeight: 0.9
```

---

Nuked